tests/tacacs_server.yaml.example

```diff
@@ -0,0 +1,6 @@
+# Tacacs server configuration for test_aaa_authorization_service.rb
+# Tacacs server must be set up with external access for test to run
+host: '10.122.197.197'
+key:  'testing123'
+vrf:  'management'
+intf: 'mgmt0'
```

Comment:
I'm thinking it might be handier to make this the *actual* `tacacs_server.yaml` file instead of `.example`, and just comment out the keys. Not a big deal if you want to keep it as is.